locallibrary/catalog/admin.py

```diff
@@ -1,3 +1,34 @@
 from django.contrib import admin
 
 # Register your models here.
+from .models import Author, Genre, Book, BookInstance
+
+admin.site.register(Genre)
+
+@admin.register(Author)
```

Comment:
có 2 cách để viết, 1 là kiểu này 2 là : admin.site.register(Author, AuthorAdmin)

Reply:
vote dùng cách viết decorator ngay trên class cho tường mình nha c
```suggestion
@admin.register(Author)
class AuthorAdmin(admin.ModelAdmin):
...

```